README.md

```diff
@@ -46,6 +46,26 @@ channel. Uncheck this box for faster deployment, provided the Roku does not cras
 `Automatically discover Rokus on the local network` - if checked, will automatically attempt to discover all Rokus on the local network.
 If un-checked, will disable automatic discovery, only allowing Roku devices to be manually entered.
 
+#### .rokudevignore
+
+Ignores can optionally be placed in the file `.rokudevignore`. This file should 
+reside in the same directory as the manifest file.
+
+Rules:
+
+- Lines beginning with a hash (`#`) will be ignored.
+- Lines beginning with a exclamation mark (`!`) will be treated as unignores.
```

Comment:
How is this supposed to work? If the unignored file/directory is subordinate to an ignored directory, then the unignored item will never get included, as the directory tree below the ignored directory will never get traversed.

Reply:
It is an oversight. I will push a commit that properly flows to the recursive call and checks ignores against the parent directories.